### PR TITLE
Ember 3.0 update

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,41 +17,47 @@
...
-    "broccoli-funnel": "^1.0.1",
-    "broccoli-merge-trees": "^1.1.1",
-    "ember-cli-babel": "^5.1.7",
-    "ember-cli-htmlbars": "^1.1.1",
+    "broccoli-funnel": "^2.0.1",
+    "broccoli-merge-trees": "^1.2.0",
+    "ember-assign-polyfill": "^2.2.0",
+    "ember-cli-babel": "^6.6.0",
+    "ember-cli-htmlbars": "^2.0.1",
     "ember-getowner-polyfill": "^1.2.2"
```
Comment (lamabiker) [inline package.json]: I believe this can be removed for Ember versions > 2.3

Comment (rwwagner90) [reply package.json]: This is an addon though, so we don't know what Ember version people will be using, so we need to keep it.

File: testem.js [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
-  ]
+  launch_in_dev: [
+    'Chrome'
+  ],
+  browser_args: {
+    Chrome: {
+      mode: 'ci',
+      args: [
+        // --no-sandbox is needed when running Chrome inside a container
+        process.env.TRAVIS ? '--no-sandbox' : null,
```
[the remark this answers is not in the document]
Comment (rwwagner90) [reply testem.js]: I don't think that will change anything. Looks like Travis just choked a bit. This is the generated config from ember-cli and we should likely keep it.

[the remark this answers is not in the document]
Comment (rwwagner90) [reply testem.js]: Can we just restart those failing travis jobs? I would but I do not have permissions.

[the remark this answers is not in the document]
Comment (ahmadsoe) [reply testem.js]: Travis restarted! It looks green now.

[the remark this answers is not in the document]
Comment (rwwagner90) [reply testem.js]: 👏 

It there anything else I need to do @ahmadsoe?

[the remark this answers is not in the document]
Comment (ahmadsoe) [reply testem.js]: No, I think we're good for now.

**Comment by ahmadsoe** [comment]
Thank you @rwwagner90

**Comment by rwwagner90** [comment]
You're welcome @ahmadsoe! Could we get a release with these changes please?

**Pull request merged**